jwk/handler.go

```diff
@@ -137,6 +135,12 @@ func (h *Handler) GetKey(w http.ResponseWriter, r *http.Request, ps httprouter.P
 		return
 	}
 
+	keys, err = FilterOpaqueKeys(keys)
```

Comment:
I would like to highlight, that opaque keys (private keys of assymetric key pairs and symmetric secret keys) cannot be returned like it was possible with software key store. How does this impact admin API? Only public keys of asymetric key pairs are returned. What about symmetric keys - should GetKey return error? What about GetKeySet where set can contain both symmetric/assymetric keys?